Ginger/GingerCoreTest/Misc/NonDriverActionTest.cs

```diff
@@ -309,7 +310,90 @@ public void RestartAgentNotRunningTest()
             Assert.AreEqual(eRunStatus.Passed, actAgentManipulation.Status, "Action Status");
             Assert.IsTrue(actAgentManipulation.ExInfo.Contains("Agent is not running"));
         }
-        
 
+        [TestMethod]
+        public void XMLRequestFromFileCheckedTestForPath()
+        {
+            //Arrange
+            ActXMLTagValidation action = new ActXMLTagValidation();
+            action.DocumentType = GingerCore.Actions.XML.ActXMLTagValidation.eDocumentType.XML;
+            action.ReqisFromFile = true;
+            action.InputFile.Value = TestResources.GetTestResourcesFile(@"XML\book.xml");
```

Comment:
do not use slash in the file paths. It will not work on linux. Please use Path.DirectorySeparator